unic/char/property/src/macros.rs

```diff
@@ -48,13 +48,44 @@
 /// # fn main() {}
 /// ```
 ///
+/// # Syntax (Binary Property)
+///
+// Disable compile because the included file does not exist
+/// ```ignore
+/// #[macro_use] extern crate unic_char_property;
+///
+/// // First we define the type itself.
+/// char_property! {
+///     /// This is the newtype used for the character property.
+///     pub struct MyProp(bool) {
+///         abbr => "AbbrPropName";
+///         long => "Long_Property_Name";
+///         human => "Human-Readable Property Name";
+///
+///         // Unlike an enumerated property, a binary property will handle the table for you.
+///         // This requires `unic_utils` to be in scope.
+///         data_table_path => "../tables/prop_values.rsv";
+///
+///         // A function that returns whether the given character has the property or not.
+///         pub fn is_prop(&self) -> bool;
+///     }
+///
+///     // A function that returns whether the given character has the property or not.
+///     pub fn is_property(char) -> bool;
+/// }
+/// ```
+///
 /// # Effect
 ///
 /// - Implements the `CharProperty` trait and appropriate range trait
 /// - Implements `FromStr` accepting either the abbr or long name, ascii case insensitive
+/// - Implements `From` to/from the wrapped type
```

Comment:
I've called whether we actually want to do this for binary properties into question below. Make sure to update this along with the decision we make.